lib/rooms.js

```diff
@@ -53,6 +53,12 @@ Room.prototype.send = function(message, cb) {
   return cb ? msg.nodeify(cb) : msg;
 };
 
+Room.prototype.sendStatus = function(message, cb) {
+  var msg = this.client.post(this.path + '/' + this.id + '/chatMessages', {body: {text: message, status: true}});
+  return cb ? msg.nodeify(cb) : msg;
```

Comment:
oh, so if the callback is undefined it'll return the promise? 

will change this in another PR